src/resources/js/store/localisation.js

```diff
@@ -6,11 +6,14 @@ export const state = {
 
 export const getters = {
     isInitialised: state => Object.keys(state.i18n).length > 0,
-    __: (state, getters, rootState, rootGetters) => (key) => {
-        const lang = rootGetters['preferences/lang'];
+    __: (state, getters, rootState, rootGetters) => (key, params) => {
+        const lang = rootGetters['preferences/lang']
+          || ((rootState.preferences && rootState.preferences.global)
+            ? rootState.preferences.global.lang : null);
+
```

Comment:
let's identify the cause for the problem you encountered and fix it at that level.

At a first sight the three lines are equivalent with

` const lang = rootGetters['preferences/lang'] || null;`

Reply:
As mentioned, the issue is with a fresh-load when localisations are set, the translations don't show up.

My investigation indicated that, for some reason, the getter returns null although the state is already filled (although they should both return the exact same in theory)

Reply:
Note that `null` should be `undefined` to avoid complications in the framework (document.title becomes null, and likely other hidden things)

Reply:
I'll try in the following days to replicate and see if what we have above is the best fix